apps/ui/src/components/__tests__/search-box.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+
+import { SearchBox } from "../search-box";
+
+describe("search-box", () => {
+  it("should render search box successfully", () => {
+    const handleSearch = jest.fn();
+
+    const { getByRole } = render(
+      <SearchBox handleSearch={handleSearch} searchValue="" />,
+    );
+
+    expect(getByRole("textbox")).toBeInTheDocument();
+    expect(getByRole("textbox")).toHaveAttribute("placeholder", "Search...");
+  });
+
+  it("should have the correct placeholder", () => {
+    const handleSearch = jest.fn();
+
+    const { getByRole } = render(
+      <SearchBox
+        handleSearch={handleSearch}
+        placeholder="test search"
+        searchValue="search text"
+      />,
+    );
+
+    expect(getByRole("textbox")).toBeInTheDocument();
+    expect(getByRole("textbox")).toHaveAttribute("placeholder", "test search");
+    expect(getByRole("textbox")).toHaveAttribute("value", "search text");
+  });
```

Comment:
I expect to see a test something like: 

"should render a list of traces, search in the box and assert the list"

Reply:
Yeah will add more test cases to make sure input is working correctly, as per rendering traces we should test `traces.tsx` itself

Reply:
Nice work it much more tested now!